src/components/banner.tsx

```diff
@@ -36,7 +36,7 @@ const Banner = () => (
   <BannerWrapper>
     <BannerText>
       <a href="/dataplatform">
```

Comment:
@michellegreer The link should be changed from `dataplatform` to `docs/data-platform/data-proxy` here in line 38